### PR TITLE
Test patch set for upstreaming

### PR DESCRIPTION
**Pull request by eukarpov**

### CODE REVIEW

File: .github/workflows/advanced.yml [hunk 1]
```diff
@@ -83,7 +82,7 @@ jobs:
     strategy:
       matrix:
         arch: [aarch64, x86_64]
-        platform: [w64-mingw32, linux-gnu]
+        platform: [linux-gnu]
```
Comment (Blackhex) [inline .github/workflows/advanced.yml:85]: We should test `x86_64-w64-mingw32` as well.

**Comment by Blackhex** [comment]
@eukarpov Can we close this one already?

**Comment by eukarpov** [comment]
> @eukarpov Can we close this one already?

@Blackhex This PR is used regularly to test new and updated patch series.

**Comment by Blackhex** [comment]
> @Blackhex This PR is used regularly to test new and updated patch series.

OK, I thought it's deprecated by this one #85.

**Comment by eukarpov** [comment]
> > @Blackhex This PR is used regularly to test new and updated patch series.
> 
> OK, I thought it's deprecated by this one #85.

Actually you are right, this PR is not needed anymore. It was used for patch series 1 which is already upstreamed.

**Pull request closed without merging**